deepmd/dpmodel/atomic_model/property_atomic_model.py

```diff
@@ -15,3 +17,24 @@ def __init__(self, descriptor, fitting, type_map, **kwargs):
                 "fitting must be an instance of PropertyFittingNet for DPPropertyAtomicModel"
             )
         super().__init__(descriptor, fitting, type_map, **kwargs)
+
+    def apply_out_stat(
+        self,
+        ret: dict[str, np.ndarray],
+        atype: np.ndarray,
+    ):
+        """Apply the stat to each atomic output.
+        In property fitting, each output will be multiplied by label std and then plus the label average value.
+
+        Parameters
+        ----------
+        ret
+            The returned dict by the forward_atomic method
+        atype
+            The atom types. nf x nloc. It is useless in property fitting.
+
+        """
+        out_bias, out_std = self._fetch_out_stat(self.bias_keys)
+        for kk in self.bias_keys:
+            ret[kk] = ret[kk] * out_std[kk][0] + out_bias[kk][0]
```

Comment:
What does the zero index represent here?

Reply:
When `fitting_net/type` is `property`. The property bias layer is used to normalize the property output of the model.i.e.,
$p_{bias} (MLP(D_i))= MLP(D_i) * std+ mean$
1. `std`: The standard deviation of the property label
2. `mean`: The average value of the property label

So the `out_bias` and `out_std` here is not related to element type. Any index is the same result.